frame/contracts/src/weights.rs

```diff
@@ -508,6 +509,17 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes((100 as Weight).saturating_mul(r as Weight)))
 	}
+	// Storage: System Account (r:1 w:0)
+	// Storage: Contracts ContractInfoOf (r:1 w:1)
+	// Storage: Contracts CodeStorage (r:1 w:0)
+	// Storage: Timestamp Now (r:1 w:0)
+	fn seal_set_code_hash(r: u32, ) -> Weight {
+		   (265_861_000 as Weight)
+			   // Standard Error: 1_212_000
+			   .saturating_add((303_302_000 as Weight).saturating_mul(r as Weight))
+			   .saturating_add(T::DbWeight::get().reads(4 as Weight))
+			   .saturating_add(T::DbWeight::get().writes(1 as Weight))
+	}
```

Comment:
The benchmark is wrong. It does not report any storage access (for `r`). Reason is that the code hashes do not exist which you are setting to.